FrontMen/fm-hours
# feat(timesheet-for-specific-employee): added getter& adjusted routing

# Changes

## Related issues

#277 

## Added

- New Getter GetEmployeeById 
- Added in the /admin/employees route a button to "manage month overview" per employee.

## Changed

- Adjust /month routing to support employee Id /month/*

## How to test

- Login as an Admin
- Go to /admin/employees
- Click "manage month overview"
- You should see the month overview of the selected user

- Try to also test the route if no ID is given, you should be redirected to /month/deafult route.

## Screenshots

<img width="1702" alt="image" src="https://user-images.githubusercontent.com/46437510/169833235-dea838e4-b7bf-469a-931f-b997a52f0b9d.png">
<img width="1714" alt="image" src="https://user-images.githubusercontent.com/46437510/169833306-8ddecc8a-61ff-4436-bf99-91eea44d9526.png">
